apps/clr-ui-tokens/src/app/accordion-demo/accordion-demo.component.html

```diff
@@ -0,0 +1,32 @@
+<h3 cds-layout="m-t:none m-b:md">Accordion</h3>
+
```

Comment:
If `apps/clr-ui-tokens/src/app/accordion-demo/accordion-demo.component.scss` is an empty file, why do we need it?